chore: Update Readme (#161)
nobl9/sloctl: README.md

<!-- markdownlint-enable line-length html -->

Sloctl is a command-line interface (CLI) for [Nobl9](https://www.nobl9.com/).
You can use sloctl to manage multiple Nobl9 configuration objects
such as [SLOs](https://docs.nobl9.com/getting-started/nobl9-resources/slo),
[Projects](https://docs.nobl9.com/getting-started/nobl9-resources/projects)
or [Alert Policies](https://docs.nobl9.com/getting-started/nobl9-resources/alert-policies).

The web user interface is available to give you an easy way to create
and update SLOs and other resources, while sloctl aims to provide a
systematic and automated approach to maintaining SLOs as code.

You can use it in CI/CD or your terminal power-user workflows :fire:

## Usage

> Sloctl comes with built-in documentation for each command, to access it,
> run:

Reviewer: ```suggestion
Sloctl includes built-in documentation for each command. To access it, run:
```

### Docker

Sloctl official images are hosted on [hub.docker.com](https://hub.docker.com/r/nobl9/sloctl).

Here's an example workflow for managing Project object:

1. Export sloctl access keys through environment variables.

```shell
export SLOCTL_CLIENT_ID=<your-client-id>
export SLOCTL_CLIENT_SECRET=<your-client-secret>
```

2. Create a sample Project object and save it to `project.yaml` file.

   ```shell
   cat << EOF > project.yaml
   apiVersion: n9/v1alpha
   kind: Project
   metadata:
     displayName: My Project
     name: my-project
   spec:
     description: Just and example Project :)
   EOF
   ```
3. Apply the project from `project.yaml`.

>    Interactive mode has to be used for `docker run` in order to keep the STDIN
>    open and allow the `project.yaml` contents to be piped into
>    `docker run` call.

Reviewer: ```suggestion
   To keep STDIN open and allow piping the contents of `project.yaml` into the `docker run` command, use interactive mode with `docker run`.
```
